src/utils.js

```diff
@@ -383,5 +383,12 @@ export function combineURLs(baseURL, relativeURL) {
  * @returns {Boolean} true if available, false otherwise.
  */
 export const isPushNotificationAvailableForUser = (state) => (
-  state.pushNotification.available && state.pushNotification.deviceRegistered
+  state.pushNotification.available
+    // On iOS a simulator can't register a device token on APNS
+    && state.pushNotification.deviceRegistered
+    // TODO: We should drop this condition when we add support other networks
```

Comment:
Do we have an issue to discuss this?

Reply:
This one: https://github.com/HathorNetwork/internal-issues/issues/285

We can keep it open until we find a definite solution. What do you think?

Reply:
Here is is a PR to track this TODO: https://github.com/HathorNetwork/internal-issues/issues/288